packages/sanity/src/desk/panes/document/documentPanel/header/ValidationMenu.tsx

```diff
@@ -1,93 +0,0 @@
-import {ErrorOutlineIcon, InfoOutlineIcon, WarningOutlineIcon} from '@sanity/icons'
-import {
-  isValidationInfoMarker,
-  isValidationWarningMarker,
-  isValidationErrorMarker,
-  Path,
-} from '@sanity/types'
-import {Button, ButtonProps, Menu, MenuButton} from '@sanity/ui'
-import React, {useCallback, useMemo, useId} from 'react'
-import {useDocumentPane} from '../../useDocumentPane'
-import {ValidationList} from 'sanity'
```

Comment:
Correct me if I am wrong but this was the only component using `ValidationList` https://github.com/sanity-io/sanity/blob/next/packages/sanity/src/core/components/validation/validationList.tsx 

and seems like `ValidationList` is the only component using `ListItem` https://github.com/sanity-io/sanity/blob/next/packages/sanity/src/core/components/validation/listItem.tsx#L24

Reply:
Thanks for being thorough! Removed those components too!